src/Box.js

```diff
@@ -4,11 +4,38 @@ import PropTypes from 'prop-types'
 
 const align = responsiveStyle('text-align', 'align')
 
+
+function boxShadow(props) {
+  const shadowColor = 'rgba(0,0,0,0.08)'
+  const shadows = [
+    `0 0 4px 0 ${shadowColor}`, 
+    `0 8px 8px 0 ${shadowColor}`, 
+    `0 16px 16px 0 ${shadowColor}`, 
+    `0 32px 32px 0 ${shadowColor}`
+  ]
+  const boxShadows = {
+    'small': {
+      'box-shadow': shadows[0]
+    },
+    'medium': {
+      'box-shadow': shadows.slice(0,2).join(", ")
+    },
+    'large': {
+      'box-shadow': shadows.slice(0,3).join(", ")
+    },
+    'xlarge': {
```

Comment:
For box shadow aliases, would you think using short-hand terms such as `sm`, `md`,  `lg`, and `xl`? This will align to the naming convention in similar aliases in theme:
https://github.com/pricelinelabs/design-system/blob/master/src/theme.js#L26